src/AccessibilityInsights.SharedUx/Dialogs/HotkeyGrabDialog.xaml.cs

```diff
@@ -12,29 +12,20 @@ namespace AccessibilityInsights.SharedUx.Dialogs
     /// </summary>
     public partial class HotkeyGrabDialog : Window
     {
-        /// <summary>
-        /// Modifier key
-        /// </summary>
         ModifierKeys ModKey;
 
-        /// <summary>
-        /// Selected non-modifier key
-        /// </summary>
         Key SelectedKey;
 
         /// <summary>
         /// Button to update
         /// </summary>
         Button ButtonParent;
 
-        /// <summary>
-        /// Constructor
-        /// </summary>
-        /// <param name="tb">Button which triggered dialog</param>
         public HotkeyGrabDialog(Button tb)
```

Comment:
When I was testing this, I was surprised by the modifier key presses being "sticky" by default; I'd have expected the sequence "ctrl down, ctrl up, shift down, x" to result in the shortcut being "shift + x" but it is actually "ctrl + shift +x".

I realize this behavior is not different from the original, so I think it's appropriate for this PR to avoid changing it, but I wonder if it's just me that was surprised by it/whether we should file an issue to consider changing it to be more in line with other programs (the first two I tried were VS and VS Code, both of which were non-sticky)